helm/featurehub/templates/dacha/ingress.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.dacha.enabled }}
+{{- if .Values.dacha.ingress.enabled -}}
```

Comment:
this is a little unusual - do you see a use case where Dacha needs an ingress? It should never be exposed outside the cluster, only MR and Edge would normally be?

Reply:
Hello!

Probably you are right, as I don't have all the insights of the application, I did my best just by reading the documentation, but if dacha having an ingress does not make sense, just let me know and I'll remove its template and the corresponding values :)